app/hardware/server/provision/provision.form.component.js

```diff
@@ -355,13 +356,19 @@
         });
     }
 
+    function openBandwidthHelpModal() {
+      var modal = $uibModal.open({
```

Comment:
I think we have a Modal service for this which would be preferred. I think you can pass it just a lang string which circumvents the need for a template file (the language file has the title and body).